spec/requirements.md

```diff
@@ -10,6 +10,7 @@ This page is for brainstorming on the technical requirements for solving our [pr
 1. Namespace management should not be any more complicated than S3 currently
 1. Performance should be competitive with existing solutions
 1. The design should be simple to implement
+1. The system should support reflection
```

Comment:
Add:
1. The system should support dynamic and lazy registration of classes and methods.